tests/Api/Admin/AdminUsersTest.php

```diff
@@ -15,11 +15,148 @@
 
 use Sylius\Component\Core\Model\AdminUserInterface;
 use Sylius\Tests\Api\JsonApiTestCase;
+use Sylius\Tests\Api\Utils\AdminUserLoginTrait;
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\Response;
 
 final class AdminUsersTest extends JsonApiTestCase
 {
+    use AdminUserLoginTrait;
+
+    /** @test */
+    public function it_allows_admin_users_to_log_in(): void
+    {
+        $this->loadFixturesFromFile('authentication/api_administrator.yaml');
+
+        $this->client->request(
+            method: 'POST',
+            uri: '/api/v2/admin/administrators/token',
+            server: self::CONTENT_TYPE_HEADER,
+            content: json_encode([
+                'email' => 'api@example.com',
+                'password' => 'sylius',
+            ], \JSON_THROW_ON_ERROR),
+        );
+
+        $response = $this->client->getResponse();
+
+        $this->assertResponse($response, 'admin/admin_user/log_in_admin_response', Response::HTTP_OK);
+    }
+
+    /** @test */
+    public function it_gets_administrators(): void
+    {
+        $this->loadFixturesFromFiles(['authentication/api_administrator.yaml', 'administrator.yaml']);
+        $header = array_merge($this->logInAdminUser('api@example.com'), self::CONTENT_TYPE_HEADER);
+
+        $this->client->request(method: 'GET', uri: '/api/v2/admin/administrators', server: $header);
+
+        $this->assertResponse(
+            $this->client->getResponse(),
+            'admin/admin_user/get_administrators_response',
+            Response::HTTP_OK,
+        );
```

Comment:
Passing `Response::HTTP_OK` is not necessary as it is a default value
```suggestion
        $this->assertResponse($this->client->getResponse(), 'admin/admin_user/get_administrators_response');
```